.teamcity/src/main/kotlin/model/CIBuildModel.kt

```diff
@@ -88,7 +88,6 @@ data class CIBuildModel(
             ),
             functionalTests = listOf(
                 TestCoverage(3, TestType.platform, Os.LINUX, JvmCategory.MIN_VERSION, DEFAULT_LINUX_FUNCTIONAL_TEST_BUCKET_SIZE),
-                TestCoverage(4, TestType.platform, Os.WINDOWS, JvmCategory.MAX_VERSION),
```

Comment:
I don't think it is a good idea to move the Windows platform tests to ready for release for two reasons:
- We often expose some locking problems on Windows when running those builds more often.
- By doing so, we don't have coverage for Platform tests on JDK 21.

Instead, we should work on making those builds faster.

Do we actually wait for them, or are other things running in parallel that take longer anyway?